rero_ils/modules/documents/jsonschemas/documents/document-minimal-v0.0.1_src.json

```diff
@@ -863,41 +1046,92 @@
         }
       }
     },
-    "series": {
-      "title": "Series",
-      "description": "Series to which belongs the resource.",
+    "seriesStatement": {
+      "title": "Series statement",
+      "description": "Series and eventual subseries to which a resource belongs (MARC 490).",
```

Comment:
Do we have to say it comes from MARC 490?

Reply:
@rerowep Yes, this description comes from the metadata group specifications. It's there to help librarian, at least the ones that can speak MARC fluently.

Reply:
Yes